### ISSUE TITLE
Callbacks are not wrapped in strongly typed interfaces

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rubal18/google-checkout-java-sample-code#1]
```
The merchant calculation callback and notification processor interfaces
currently take Strings and return Strings.
I am planning to wrap these in strongly types interfaces - very soon!  

S...
```

Original issue reported on code.google.com by `si...@simonjsmith.com` on 5 Apr 2007 at 9:42

**Comment by GoogleCodeExporter** [comment]
Original comment by `simonjsm...@gmail.com` on 5 Apr 2007 at 9:46
- Changed state: **Started**
- Added labels: **Priority-High**, **Type-Enhancement**
- Removed labels: **Priority-Medium**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.3
```

Original comment by `simonjsm...@gmail.com` on 17 Jun 2007 at 1:33
- Changed state: **Fixed**
